src/mdio/converters/segy.py

```diff
@@ -291,7 +291,7 @@ def populate_non_dim_coordinates(
     return dataset, drop_vars_delayed
 
 
-def _get_horizontal_coordinate_unit(segy_headers: list[Dimension]) -> LengthUnitModel | None:
+def get_horizontal_coordinate_unit(segy_headers: list[Dimension]) -> LengthUnitModel | None:
```

Comment:
```suggestion
def _get_horizontal_coordinate_unit(segy_headers: list[Dimension]) -> LengthUnitModel | None:
```

I don't think we want to denote this as a public function.